src/collections/vec.rs

```diff
@@ -1896,6 +1896,16 @@ impl<'bump, T: 'bump> Extend<T> for Vec<'bump, T> {
     }
 }
 
+impl<'bump, T> FromIteratorIn<T> for Vec<'bump, T> {
+    type Alloc = &'bump Bump;
+    fn from_iter_in<I>(iter: I, alloc: Self::Alloc) -> Self
+    where
+        I: IntoIterator<Item = T>,
+    {
+        Vec::from_iter_in(iter, alloc)
+    }
+}
+
```

Comment:
Can you avoid modifying the files that are copied from `std`? This makes it easier to update these files in the future.

Reply:
Oh, I didn't realize these were straight from `std`. I'll move the impls back to their original place, sorry!

Reply:
Oh, I didn't realize these were straight from `std`. I'll move the impls back to their original place, sorry!